worker/jobTypes/githubJob.js

```diff
@@ -268,6 +318,36 @@ class GitHubJobClass {
                 'ERROR: makefile does not exist in /makefiles directory on meta branch.'
             );
         }
+        //set up env vars for all jobs
+        const pathPrefix = await this.writeEnvFile(isProdDeployJob)
```

Comment:
should only run for next gen jobs

Reply:
got it - I mistakenly thought we only published next-gen